gulpfile.js

```diff
@@ -68,46 +60,62 @@ function lint() {
     .pipe(gjslint.reporter('console'))
 }
 
-function bundle(browserify, env) {
-  return browserify
-    .transform(babelify)
-    .transform(envify, env)
-    .bundle()
-    .on('error', gutil.log.bind(gutil, 'Browserify Error'))
-    .pipe(source(jsFileName))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-      .pipe(uglify({
-        preserveComments: 'some'
-      }))
-      .on('error', gutil.log)
-    .pipe(sourcemaps.write('./'))
-    .pipe(gulp.dest('dist'));
+function bundle(format) {
+  return esperanto.bundle({
+    base: '.',
+    entry: entryFileName,
+  }).then(function(bundle) {
+    return bundle[format]({
+      strict: true,
+      sourceMap: 'inline',
```

Comment:
Ah, okay. I try to avoid inline src maps, 'cause if I remember correctly it causes the user to download bytes they otherwise won't need to. If an external map is used, then it's only downloaded when dev tools is open.